test/slider-test.jsx

```diff
@@ -19,147 +25,149 @@ function testKeys(slider, tests) {
 
 const newSlider = props => new Slider({ ...Slider.defaultProps, ...props });
 
-describeWithDOM('<Slider />', () => {
-  describe('render', () => {
```

Comment:
I removed the `describeWithDOM` wrapper above. I could just add another `describe` block here instead? although it doesn't really seem necessary...